index.Rmd

```diff
@@ -18,7 +18,7 @@ knitr::opts_chunk$set(echo = TRUE)
 # Nicole's New section
 
 - this is where Nicole entered data
-- this is where Nicole entered more data
+- this is where Nicole entered even more!
```

Comment:
I'm not sure about this new data set.

Reply:
ok, I will take some of the newness away